Lombiq.NodeJs.Extensions/scripts/handle-error.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @summary Helper functions to display errors in the format MSBuild can understand.
+ */
+
+const os = require('os');
+
+function handleErrorObjectInner(error, type, defaultCode) {
+    if (error === null || error === undefined) {
+        return handleErrorObjectInner(Error("Missing error argument"), 'error', 'META-ERROR');
+    }
+
+    const code = error.code || defaultCode;
+    const path = error.path || 'no-path';
+    const message = (error.message || error)?.toString().replace(/^Error[ :]+/i, '');
```

Comment:
I would keep `error` all lowercase for consistency with the rest of the regex:
```suggestion
    const message = (error.message || error)?.toString().replace(/^error[ :]+/i, '');
```